reference/openapi.yaml

```
@@ -378,6 +383,31 @@ paths:
     post:
       $ref: "./paths/webhooks__inbound_webhook.yaml"
 
+  "/flows/{flow}":
+    parameters:
+      - $ref: "./path-parameters/flow.yaml"
+    get:
+      $ref: "./paths/flow__list-rules.yaml"
+
+  "/flows/{flow}/actions/{action}/rules/":
+    parameters:
+      - $ref: "./path-parameters/flow.yaml"
+      - $ref: "./path-parameters/flow_action.yaml"
+    post:
+      $ref: "./paths/flow__add-rule.yaml"
+
+  "/flows/{flow}/actions/{action}/rules/{rule_id}":
```

```suggestion
  "/flows/{flow_id}/actions/{action_id}/rules/{rule_id}":
```

Same question here, should it be `flow` and `action` or `flow_id` and `action_id`. The latter would be more consistent.